app/controllers/products_controller.rb

```diff
@@ -75,14 +63,17 @@ def product_params
       :description,
       :item_number,
       :vendor,
-      :vendor_link,
       :duns,
       :shipping_timeline_estimate,
-      :price,
```

Comment:
we'll want `:vendor_link` and `:price` here

Reply:
ah yea I think I missed that in the rebase, thanks for catching